packages/react-components/src/templates/EventsCalendar.tsx

```diff
@@ -13,10 +13,31 @@ type EventsCalendarPageProps = {
   calendars: ReadonlyArray<CalendarResponse>;
 };
 
-const EventsCalendarPage: React.FC<EventsCalendarPageProps> = (props) => (
+const EventsCalendarPage: React.FC<EventsCalendarPageProps> = ({
+  calendars,
+}) => (
   <div css={containerStyles}>
-    <GoogleCalendar {...props} />
-    <CalendarList page="calendar" {...props} />
+    <GoogleCalendar calendars={calendars} />
+    <CalendarList
+      calendars={calendars.filter(
+        (calendar) =>
+          calendar.groups?.some(({ active }) => active) ||
```

Comment:
Why are groups potentially undefined? Should always be an array even if its an empty one